config/locales/fr.yml

```
@@ -563,9 +563,9 @@ fr:
       execute_confirmation: Voulez-vous vraiment appliquer ce déplacement ?
       frame_table_title: Baies après déplacements
       edit_connections: Modifier les connexions
-      server: Matériel
+      number_u: N° U
```

```suggestion
      position: Position
```

May be use Server locale for this.